velox/docs/functions/coverage.rst

```diff
@@ -216,7 +216,7 @@ Here is a list of all scalar and aggregate Presto functions with functions that
     :func:`array_join`                        :func:`e`                                 json_size                                 sha1                                      st_y                                          :func:`checksum`
     :func:`array_max`                         :func:`element_at`                        :func:`least`                             :func:`sha256`                            st_ymax                                       classification_fall_out
     :func:`array_min`                         :func:`empty_approx_set`                  :func:`length`                            :func:`sha512`                            st_ymin                                       classification_miss_rate
-    array_normalize                           enum_key                                  levenshtein_distance                      shuffle                                   :func:`strpos`                                classification_precision
```

Comment:
This change should not be part of this PR. After the PR lands, we'll need to update coverage maps separately following instructions in velox/functions/prestosql/coverage/README.md